Unable to use a CustomService in the way it's described in the docs.

## The problem

I wrote a CustomService according to the [doc](http://webdriver.io/guide/testrunner/services.html) and assigned that to the services property.

## Environment

* WebdriverIO version: 4.13.2
* Node.js version:v8.3.0

## Details

 I used CommonJS syntax for import's/export's. As the result I got message:
`
SyntaxError: Invalid regular expression: /Cannot find module 'wdio-class LocalConnectionService {
    /**
     * @return {Promise}
     */
    onPrepare({localConnection}) {
        return new Promise(function(resolve, reject) {
            cbt.start(
                localConnection,
                function(err) {
                    if (!err) {
                        resolve();
                    } else {
                        reject(err);
                    }
                });
        });
    }
    /**
     * @return {Promise}
     */
    onComplete() {
        return new Promise((resolve, reject) => {
            cbt.stop((err) => {
                if (err) {
                    reject();
                }
                resolve();
            });
        });
    }
}-service/launcher'/: Nothing to repeat
` 

Then, I browsed the source code of wdio and found the [function](https://github.com/webdriverio/webdriverio/blob/ba5ccb51c3113367bc9295cbcc28b5a18d902c0d/lib/launcher.js#L496) getLauncher.

There are the 'if' statement where service's type is checked. 
``` 
 if (typeof serviceName === 'object') {
      launchServices.push(serviceName)
      continue
 }
```

Moving on to running a service.
There is the function [runServiceHook](https://github.com/webdriverio/webdriverio/blob/ba5ccb51c3113367bc9295cbcc28b5a18d902c0d/lib/launcher.js#L194)
There is the other check which implies that service must be an object or a class with static methods (usual methods are added to a prototype)
```
if (typeof service[hookName] === 'function') {
    return service[hookName].apply(service, args);
}
```

So, I'd suggest changing the doc.

## Code To Reproduce Issue [ Good To Have ]

wdio.conf.js
`
class CustomService {
    onPrepare() { console.log('Hello, Webdriver!') }
} 

module.exports = {
    services: [CustomService],
    // Other config's props
}
`

--- a/docs/guide/testrunner/services.md
+++ b/docs/guide/testrunner/services.md
@@ -33,7 +33,7 @@ var CustomService = require('./service/my.custom.service');
 
 exports.config = {
     // ...
-    services: [CustomService],
+    services: [new CustomService()],
     // ...
 };
 ```

